orchestrator-salt/src/main/resources/salt-common/salt/monitoring/scripts/logging-agent.sh.j2

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+if [[ -f "/var/run/cdp-logging-agent/cdp-logging-agent.pid" ]]; then
+  platform="{{ salt['pillar.get']('fluent:platform') }}"
+  logging_types=""
+{%- if salt['pillar.get']('fluent:cloudStorageLoggingEnabled') %}
+  if [[ "$platform" == "AWS" ]]; then
+    logging_types="s3"
+  elif [[ "$platform" == "AZURE" ]]; then
+    logging_types="abfs"
+  elif [[ "$platform" == "GCP" ]]; then
+    logging_types="gcs"
+  fi
+  use_cloud_storage="true"
+{%- else %}
+  use_cloud_storage="false"
+{%- endif %}
+{%- if salt['pillar.get']('fluent:dbusMeteringEnabled') %}
+  logging_types="$logging_types metering_databus"
+{%- endif %}
+{%- if salt['pillar.get']('fluent:dbusClusterLogsCollection') %}
+  logging_types="$logging_types databus_service_logs"
+{%- endif %}
+  if [[ "$logging_types" != "" ]]; then
```

Comment:
if this is false then this script doesn't make anything, so I wouldn't even create the script and the cronjob if this is false and remove this condition.